sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/Storage.java

```diff
@@ -676,7 +677,13 @@ synchronized void getTokenAndCallCosmosDbApi(
             TokenExchangeServiceCallback callback) {
         TokenResult cachedTokenResult = TokenManager.getInstance().getCachedToken(partition);
         if (cachedTokenResult != null) {
-            callback.callCosmosDb(cachedTokenResult);
+            int cachedTokenResultStatusCode = Integer.parseInt(cachedTokenResult.status());
+            if (cachedTokenResultStatusCode == 200) {
```

Comment:
Why are we changing back to hardcoded 200 int now? I thought I saw something cleaner in previous iterations.

Reply:
I don't know what you're looking at, this is outdated...

Reply:
No idea, thought I clicked on files and just looking at diffs.